src/CheckPoint/CheckPointParser.cpp

```diff
@@ -10,45 +10,64 @@ CheckPointParser::CheckPointParser(b2World *world, float scale)
         : world_(world), scale_(scale) {
 }
 
-std::vector<CheckPointPtr> CheckPointParser::ParseFile(const std::string &file) {
+std::vector<CheckPointPtr>
+CheckPointParser::ParseFile(const std::string &file) {
```

Comment:
Nie wiem, po co tak te linie łamiesz. 120 znaków na pewno nie przekraczają. A piszę tutaj, bo to złamanie mi się wyjątkowo nie podoba